pil_analyzer/src/pil_analyzer.rs

```diff
@@ -189,8 +193,20 @@ impl<T: FieldElement> PILAnalyzer<T> {
                     Some(definition),
                 );
             }
-            PilStatement::ConstantDefinition(_start, name, value) => {
-                self.handle_constant_definition(name, self.evaluate_expression(&value).unwrap())
+            PilStatement::ConstantDefinition(start, name, value) => {
+                // Check it is a constant.
+                self.evaluate_expression(value.clone())
+                    .unwrap_or_else(|err| {
+                        panic!("Could not evaluate constant for namespace {name}: {value}: {err}")
+                    });
+                println!("Constant: {name}");
```

Comment:
```suggestion
```

I think we never want to use `println!()`, no? Maybe we can add a linter rule?

Reply:
good idea!

Reply:
#664